Automattic/wp-calypso
# Woo Installer: Add business-info step

#### Changes proposed in this Pull Request

* Adds business-info step to WooCommerce installation flow, that asks for product types, counts, and selling venues.
* We'll need to iterate on styling, there are differences in how labels for dropdowns and input fields are styled.

#### Testing instructions



* Go to http://calypso.localhost:3000/woocommerce-installation/
* Choose a test site site
* Click set up my store
* You should be taken to the business-info step



Fixes #57422 .
